src/siteconfig/tests/test_views.py

```diff
@@ -74,3 +78,66 @@ def testSiteConfigUpdate_uses_newly_saved_cache_data(self):
         # After calling SiteConfig.get(), cache should not be equal to the old cache
         # Comparing the `site_name` since Django's Model.__eq__ is comparing `pk`
         self.assertNotEqual(old_cache.site_name, cache.get(SiteConfig.cache_key()).site_name)
+
+    def test_SiteConfigForm_basic_tests(self):
+        """ 
+            Basic test for SiteConfigForm
+        """ 
+        owner_user = self.config.deck_owner
+        admin_user = User.objects.get(username=settings.TENANT_DEFAULT_ADMIN_USERNAME)
+        staff_user = baker.make(User, is_staff=True,)
+
+        URL = reverse("config:site_config_update_own")
+
+        def generate_form_data(**kwargs):
+            """ 
+                Helper func to generate valid form data using self.config values in order to use self.client.post()
+            """
```

Comment:
Clever!

Reply:
This func could definitely be genericized and I would love to work on it if its posted as an issue